src/collections/_documentation/development/sdk-dev/event-payloads/exception.md

```diff
@@ -166,36 +179,42 @@ information.
 
 ## Examples
 
-The following examples illustrate payloads that may be sent by SDKs in various
-circumstances.
+The following examples illustrate multiple ways to send exceptions. Each example
+contain the exception part of the [event payload]({%- link
+_documentation/development/sdk-dev/event-payloads/index.md -%}) and omit other
+attributes for simplicity.
 
 A single exception:
 
 ```json
 {
   "exception": {
-    "type": "ValueError",
-    "value": "My exception value",
-    "module": "__builtins__",
-    "stacktrace": {},
+    "values": [
+      {
+        "type": "ValueError",
+        "value": "my exception value",
+        "module": "__builtins__",
+        "stacktrace": {}
+      }
+    ]
   }
 }
 ```
 
-A list of exceptions:
+Chained exceptions:
 
 ```json
 {
   "exception": {
     "values": [
       {
         "type": "Exception",
-        "value": "Wrapped exception value",
+        "value": "initial exception",
         "module": "__builtins__"
       },
       {
         "type": "ValueError",
-        "value": "Original Exception value",
+        "value": "wrapped exception",
```

Comment:
This is the wrapping, outer exception. `Exception` is the wrapped one:

```
ValueError (caused by Exception)
```

Should we maybe clarify this differently?

Reply:
I think this would work. There is still some ambiguity in "chained exception" but the "initial" makes it clear.